src/main/java/com/google/sps/classes/SqlConstants.java

```diff
@@ -36,4 +36,26 @@
   public static final int QUESTION_FETCH_MAXID_COLUMN = 1;
   public static final int FOLLOWER_INSERT_QUESTIONID_COLUMN = 1;
   public static final int FOLLOWER_INSERT_ASKERID_COLUMN = 2;
+
+  // Constants to get user ID.
+  public static final int GET_USER_ID_EMAIL = 1;
+  
+  // Constants to add new user.
+  public static final int ADD_NEW_USER_FIRST_NAME = 1;
```

Comment:
Curious - these ints can be used on retrieval too, right? They're used in your PR for adding to the database, but maybe these names are a bit misleading if they can also be used to retrieve. Might be worth being consistent with the constants above (FOLLOWER_INSERT_ASKERID_COLUMN, for instance)

Reply:
Oh, I hadn't noticed the format of other columns, so I modified that so they are all consistent. And no, insertion and retrieval are different.